docs/tests.md

```diff
@@ -1,45 +1,81 @@
 # Running Python unit/integration tests/smoke tests
 
+## Running a single test
+
 To run a specifc unit test
 
     pytest -v PathToPythonTestFile::TestClassName::TestMethodName
-    pytest -v ./tests/neuro-san/internals/graph/test_sly_data_redactor.py::TestSlyDataRedactor::test_assumptions
+    pytest -v ./tests/neuro_san/internals/graph/test_sly_data_redactor.py::TestSlyDataRedactor::test_assumptions
 
-To run all unit tests
-    
-    You will need to start server service 1st as the test case "music_nerd_pro" [HTTP] is part of unit test.
-    build_scripts/server_start.sh
+## Different kinds of tests
 
-    pytest -v ./tests
+Neuro SAN has different kinds of tests marked with @pytest.mark.<some_marker>.
+The markers we use come in a few different flavors for different purposes.
```

Comment:
Explain that there are different kinds of tests that we use and what they are and when they are run during builds.